<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>348</width>
    <height>249</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{
background-color: qconicalgradient(cx:0.505773, cy:0.517, angle:317.8, stop:0.0852273 rgba(65, 103, 169, 255), stop:1 rgba(255, 255, 255, 255));
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="styleSheet">
       <string notr="true">#label_3{
font: 12pt &quot;Matura MT Script Capitals&quot;;
color: #000000;
background:#3a09e8;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">#label{
font: 12pt &quot;Matura MT Script Capitals&quot;;
color: #000000;
background:#3a09e8;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">#label_2{
font: 12pt &quot;Matura MT Script Capitals&quot;;
color: #000000;
background:#3a09e8;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>Time</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="styleSheet">
       <string notr="true">#label_4{
font: 12pt &quot;Matura MT Script Capitals&quot;;
color: #000000;
background:#3a09e8;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>URL or Local File</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">#pushButton {
font: 20pt &quot;Matura MT Script Capitals&quot;;
color: #000000;
background: #3a09e8;
border-radius: 10px;
}

#pushButton:hover {
font: 20pt &quot;Matura MT Script Capitals&quot;;
background: #2d7391;
color: #000000;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>ok</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">#pushButton_2 {
font: 20pt &quot;Matura MT Script Capitals&quot;;
color: #000000;
background: #3a09e8;
border-radius: 10px;
}

#pushButton_2:hover {
font: 20pt &quot;Matura MT Script Capitals&quot;;
background: #2d7391;
color: #000000;
border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>cansel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>348</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
